src/js/view/controls/controlbar.js

```diff
@@ -25,23 +25,38 @@ define([
         return element;
     }
 
+    function textIcon(name, role) {
+        const element = document.createElement('div');
+        element.className = 'jw-icon jw-icon-inline jw-text jw-reset ' + name;
+        if (role) {
+            element.setAttribute('role', role);
+        }
+        return element;
+    }
+
+    function spacer() {
+        const element = document.createElement('div');
+        element.className = 'jw-reset jw-spacer';
+        return element;
+    }
+
     function menu(name, ariaText) {
         return new Menu(name, ariaText);
     }
 
     function createCastButton(castToggle, localization) {
-        if (Browser.chrome && OS.iOS) {
+        if (!Browser.chrome || OS.iOS) {
```

Comment:
Thank you for fixing that! Wouldn't this prevent the chromecast button from appearing in the chrome iOS app?